python/fusion_engine_client/analysis/analyzer.py

```diff
@@ -1577,22 +1590,42 @@ def plot_events(self):
             return
 
         # Read the data.
-        result = self.reader.read(message_types=[EventNotificationMessage], remove_nan_times=False, **self.params)
-        data = result[EventNotificationMessage.MESSAGE_TYPE]
+        result = self.reader.read(message_types={MessageType.EVENT_NOTIFICATION} | COMMAND_MESSAGES | RESPONSE_MESSAGES,
```

Comment:
Are we expecting to ever have `COMMAND_MESSAGES` in the log? I guess we talked about that for the primary heading device sending data to the secondary, but is that what we'd want to display in this table?

Reply:
Typically not. Generally, the device will not echo the command directly back to a diagnostic interface. It will only send a `COMMAND` event notification. So in practice, this shouldn't happen under normal circumstances.

Given that, I figured it might be useful to include it here just in case the log _does_ somehow have commands. Useful for debugging, or if users are doing something custom for logging or generating p1log files.